scripts/import_libcbor.py

```diff
@@ -0,0 +1,139 @@
+import os
+import tempfile
+import shutil
+import subprocess
+import argparse
+
+def parse_version(version_string):
+    if version_string.startswith('v'):
+        version_string = version_string[1:]
+    parts = version_string.split('.')
+    if not parts:
+        raise ValueError("Invalid version string")
+    major = int(parts[0])
+    minor = 0
+    patch = 0
+    if len(parts) > 1:
+        minor = int(parts[1])
+
+    if len(parts) > 2:
+        patch = int(parts[2])
+
+    return major, minor, patch
+
+argument_parser = argparse.ArgumentParser(
+    description="Helper to import libcbor as external dependency.")
+
+argument_parser.add_argument("--version",
+                                default="v0.11.0", help="Version string to import")
```

Comment:
don't have a default value.
someone is going to run this in the future and accidentally downgrade us
```suggestion
                             required=True,
                             help="Version to import (e.g. v0.11.0)")
```